packages/roosterjs-editor-adapter/lib/editor/EditorAdapter.ts

```diff
@@ -114,10 +114,10 @@ export class EditorAdapter extends Editor implements ILegacyEditor {
 
     /**
      * Creates an instance of Editor
-     * @param contentDiv The DIV HTML element which will be the container element of editor
```

Comment:
I suggest we don't change the name here since the physical/logical root should be an internal concept. When creating editor, we should just know one single content Div.

Reply:
Makes sense! Updated PR to make the changes